src/log4net/Layout/Pattern/PatternLayoutConverter.cs

```diff
@@ -94,17 +94,12 @@ public virtual bool IgnoresException
     /// </summary>
     /// <param name="writer"><see cref="TextWriter" /> that will receive the formatted result.</param>
     /// <param name="state">The state object on which the pattern converter should be executed.</param>
-    protected internal override void Convert(TextWriter writer, object state)
+    public override void Convert(TextWriter writer, object state)
     {
-      LoggingEvent loggingEvent = state as LoggingEvent;
-      if (loggingEvent != null)
-      {
+      if (state is LoggingEvent loggingEvent)
```

Comment:
braces please - if you're using Rider or ReSharper, you can alt-enter on the `if` to get options - when you find the one to apply braces style, look at the menu to the right and "apply in file" (i'm normally too cautious to apply on a larger scale, but this beats alt-entering everything in the file)